srcpkgs/kitty/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'kitty'
 pkgname=kitty
-version=0.10.1
+version=0.11.1
 revision=1
 python_version=3
-hostmakedepends="ncurses pkg-config python3-devel wayland-devel wayland-protocols"
+hostmakedepends="ncurses pkg-config python3-devel wayland-devel wayland-protocols python3-Sphinx"
```

Comment:
`python3-devel wayland-devel wayland-protocols` dont need to be always present in the hostmakedepends, only for crossbuilding, you could add them conditionally like this
```
if [ -n "$CROSS_BUILD" ]; then
	hostmakedepends+=" python3-devel wayland-devel wayland-protocols"
fi
```